ethereum/README.md

```diff
@@ -28,7 +28,11 @@ For each test run:
 
 Run `npx ganache-cli --deterministic --time "1970-01-01T00:00:00+00:00"` to start a chain.
 
-Run the tests using `npm run test`
+Run the all ethereum tests using `npx truffle test`
+
+Run a specific test file using `npx truffle test test/wormhole.js`
+
+Run a specific test file while skipping compile `npx truffle test test/wormhole.js --compile-none`
```

Comment:
Bruce shared these with me, so I took a go at adding them for the next person so they can better understand how to run isolated tests and skip the compile overhead.